### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by derrinbeek** [Overv/TrayLightControl#2]
This program is not working for my i tried everything and it doesn't work i tried a another app named jasper goed to colorland and that app can connect but it will do the screen color and i want this app can you healp me with this issue?

Regards,
Derrin616

**Comment by derrinbeek** [comment]
I saw that the v6 version doesn't work i think i have a v6 can you please fix that problem?

**Comment by Overv** [comment]
No, because I don't have it.

**Comment by derrinbeek** [comment]
Can you please update it i want this program it looks so nice and handy and all the other programs doesn’t work

**Comment by Overv** [comment]
I'm sorry and I feel your pain, but I no longer use these lights myself.

**Comment by derrinbeek** [comment]
😒 how much do you need to edit to update it to v6?

**Comment by Overv** [comment]
I'm not sure but it's hard to develop something that I cannot test myself. Discussing with someone else all the time will have too much communication overhead and I also don't have time for this project right now.

**Comment by derrinbeek** [comment]
Ok rip do you have any other options or programs that works?

**Comment by Overv** [comment]
I used the RGBLedController app on my phone. You may be able to run it on your PC in the Android emulator.

**Comment by derrinbeek** [comment]
Yes i already thought at that but bluestacks takes too much time to start
